server/auth/types/claims_test.go

```diff
@@ -221,8 +221,22 @@ func TestGetUserInfoGroups(t *testing.T) {
 		httpClient = &HttpClientMock{StatusCode: 200, Body: body}
 
 		claims := &Claims{}
-		groups, err := claims.GetUserInfoGroups("Bearer fake", "https://fake.okta.com", "/user-info")
-		assert.Equal(t, groups, []string{"Everyone"})
+		groups, err := claims.GetUserInfoGroups("Bearer fake", "https://fake.okta.com", "/user-info", "")
+		assert.Equal(t, []string{"Everyone"}, groups)
+		assert.Equal(t, nil, err)
+	})
+}
+
+func TestGetUserInfoGroup(t *testing.T) {
```

Comment:
Am I missing something or this test identical to the one above?

Reply:
Looks like this one passes `"group"` in, I assume as part of [the third fix/feature](https://github.com/argoproj/argo-workflows/pull/11777/files#r1357568124). Could probably be done within the same suite and should be named differently etc